File: conference/management-bc/web/src/main/java/com/microsoft/conference/management/commands/AddSeatType.cs
using System;
using ENode.Commanding;

namespace ConferenceManagement.Commands
{
    public class AddSeatType : Command<Guid>
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }

        public AddSeatType() { }
        public AddSeatType(Guid conferenceId) : base(conferenceId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/management/commands/AddSeatType.java
package com.microsoft.conference.common.management.commands;

import org.enodeframework.commanding.Command;

import java.math.BigDecimal;

public class AddSeatType extends Command<String> {
    public String name;
    public String description;
    public BigDecimal price;
    public int quantity;

    public AddSeatType() {
    }

    public AddSeatType(String conferenceId) {
        super(conferenceId);
    }
}
